test/integration/app_test.exs

```diff
@@ -0,0 +1,67 @@
+defmodule Integration.AppTest do
```

Comment:
There are already some of this tests in `doctest` for Slime module, we probably should put it in one place.
https://github.com/slime-lang/slime/blob/master/lib/slime.ex#L22 & https://github.com/slime-lang/slime/blob/master/test/slime_test.exs#L3